### PR TITLE
create DataBase Tables

### PR DESCRIPTION
**Pull request by rawand1998**

### CODE REVIEW

**Review by mohammedsalah7 — APPROVED** [review]

File: server/database/config/data.sql [hunk 1]
```diff
@@ -0,0 +1,38 @@
+BEGIN;
+DROP TABLE IF EXISTS users;
```
Comment (NourAdawi) [inline server/database/config/data.sql:2]: DROP TABLE IF EXISTS users, category, product, item, cart ;

**Review by muhammadharoun — APPROVED** [review]

File: server/database/config/data.sql [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+     email VARCHAR(100) NOT NULL,
```
Comment (NourAdawi) [inline server/database/config/data.sql:6]: email VARCHAR(100) UNIQUE NOT NULL,

File: server/database/config/data.sql [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    material text
+);
+CREATE TABLE item (
+    id SERIAL PRIMARY KEY,
+      product_id  integer  REFERENCES product(id),
+    imgUrl text,
+    color text default('gray')
+);
+CREATE TABLE cart (
+    user_id integer REFERENCES users(id),
```
Comment (NourAdawi) [inline server/database/config/data.sql:34]: user_id integer   PRIMARY KEY REFERENCES users(id)

File: server/database/config/data.sql [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    description text NOT NULL,
+    rate integer,
+    category_id  integer REFERENCES category(id),
+    material text
+);
+CREATE TABLE item (
+    id SERIAL PRIMARY KEY,
+      product_id  integer  REFERENCES product(id),
+    imgUrl text,
+    color text default('gray')
```
Comment (NourAdawi) [inline server/database/config/data.sql:31]: color text  DEFAULT "gray"

**Review by NourAdawi — CHANGES_REQUESTED** [review]

**Pull request merged**